### ISSUE TITLE
Channel routing prioritization

### ISSUE THREAD

**Issue by JustanotheruserNick** [AppliedEnergistics/Applied-Energistics-2#7496]
### Describe the bug

I have uploaded a video to showcase this issue because it's a bit hard to explain.

https://youtu.be/qBkDqefhP8c

Basically, I am having issues with channels not being routed properly. The channel should go to the controller via the shortest way possible, which it does until you add a p2p node. As soon as you add a p2p node it pushes the channels to every cable. Also yes, I've tried to add the other sides of the p2p tunnes to the controller, still no luck (https://i.imgur.com/4e9JtG0.png) This is still happening, this cable should be 0 channels: https://i.imgur.com/3L5qqOR.png

This problem first came up in my server with my friends. I made a single player world to see if it was some wiring messup in my base but it turns out that it's probably a bug. Please let me know if it's acting as intended or not. I haven't played any newer version of AE2 since I usually play revelations.

### How to reproduce the bug

Adding a p2p tunnel to a controller will make the rest of the cables that are attached to the p2p tunnel not route channels properly

### Expected behavior

I have found a video on youtube of the expected behavior:

https://youtu.be/uU9_GERV75A?t=281

### Additional details

I am using versions:
MC - 1.20.1
Forge - 47.1.3
applied energistics 2 - 15.0.15

Not sure if I needed to provide a manual crash log (F3+c) but I did anyway.

### Which minecraft version are you using?

1.20

### On which mod loaders does it happen?

Forge

### Crash log

https://pastebin.com/8ewA90b7

**Comment by Sea-Kerman** [comment]
Not a bug. That’s how controller-less networks have always worked afaik. Make a Guide (put a book in a charger) and find the page about cables, in the items-blocks-machines section. It should describe what is happening in the section about smart cable.

**Comment by Technici4n** [comment]
Indeed, you created a separate network which doesn't have a controller, so there is no pathfinding anymore.

**Comment by JustanotheruserNick** [comment]
You guys are right! I totally forgot that I need to add a controller on each section that comes off of the controller. That is my bad, thanks for the help!
